tests/wamr-test-suites/wasi-test-script/run_wasi_tests.sh

```diff
@@ -14,6 +14,37 @@ readonly WAMR_DIR="${WORK_DIR}/../../../.."
 readonly IWASM_CMD="${WORK_DIR}/../../../../product-mini/platforms/${PLATFORM}/build/iwasm"
 readonly WAMRC_CMD="${WORK_DIR}/../../../../wamr-compiler/build/wamrc"
 
+run_aot_tests () {
+    tests=$1
+    for test_wasm in ${tests}; do
+        test_aot="${test_wasm%.wasm}.aot"
+        test_json="${test_wasm%.wasm}.json"
+
+        if [ -f ${test_wasm} ]; then
+            expected=$(jq .exit_code ${test_json})
+        fi
+
+        echo "Compiling $test_wasm to $test_aot"
+        ${WAMRC_CMD} --enable-multi-thread ${target_option} \
+            -o $test_aot $test_wasm
```

Comment:
Had better use `${test_wasm}` and `${test_aot}`?

Reply:
Fixed